components/Common/AvatarGroup/avatar/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+import Image from 'next/image';
+import styles from './index.module.css';
+import type { FC } from 'react';
+
+type AvatarProps = {
+  src: string;
+};
+
+const Avatar: FC<AvatarProps> = ({ src }) => {
+  const [url, setUrl] = useState(src);
+
+  return (
+    <Image
```

Comment:
should we use `Avatar` here ? https://www.radix-ui.com/primitives/docs/components/avatar 

it has built in support for things like Fallback

Reply:
Yes, imho we should. I feel that Augustin didn't read the issue as a whole (or maybe just didn't notice them) + didn't read the latest changes to the collab guides...

Reply:
I read but I started the work in computer class and then finished at home. And so I forgot about this. _oops_ 😅

Reply:
If possible, open PRs as ready after going through your code twice or 3 times. I know sometimes we don't do that (I myself often rush with things, but when I do they are usually hot fixes)

For these sort of PRs, even tho as they are super small, focus on re-reading and reviewing your code yourself before opening a PR. Be mindful of the reviewers time.

Reply:
Thanks for mentioning it, I'll be careful next time.